cmd/server/main.go

```diff
@@ -38,6 +38,8 @@ import (
 	"github.com/google/exposure-notifications-verification-server/pkg/ratelimit"
 	"github.com/google/exposure-notifications-verification-server/pkg/ratelimit/limitware"
 	"github.com/google/exposure-notifications-verification-server/pkg/render"
+	"github.com/opencensus-integrations/redigo/redis"
+	"go.opencensus.io/stats/view"
```

Comment:
Move these to the last import block

Reply:
Done, refactored to proper import block and then sorted the imports